packages/communication-react/review/communication-react.api.md

```diff
@@ -2056,6 +2069,62 @@ export interface TypingIndicatorStylesProps extends BaseCustomStyles {
     typingUserDisplayName?: IStyle;
 }
 
+// @beta (undocumented)
+export const UPLOAD_CANCELLED_EVENT = "uploadCancelled";
```

Comment:
Why are these events separate constants rather than a single string literal union: `export const FileUploadEvent = 'cancelled' | 'completed' | 'failed' | ...`. That is more in line with the rest of our API.

Reply:
constants for avoiding spelling mistakes when using these events with event emitter as emitter can not be typed.
We do export `UploadedFileEvents` which is a string literal to be in line with our API